### PR TITLE
[Storage] Adding 2018-03-01-preview and support Management Policy

### PR DESCRIPTION
**Pull request by blueww**

### CODE REVIEW

**Comment by Tiano2017** [comment]
Looks good.

**Comment by blueww** [comment]
Would you please help to merge it?
Thanks!

File: tests/2018-03-01-preview/Microsoft.Storage.tests.json [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+    },
+    {
+      "name": "storageAccountsManagementPolicy",
+      "definition": "https://schema.management.azure.com/schemas/2018-03-01-preview/Microsoft.Storage.json#/resourceDefinitions/storageAccounts_managementPolicies",
+      "json": {
+        "type": "Microsoft.Storage/storageAccounts/managementPolicies",
+        "apiVersion": "2018-03-01-preview",
+        "name": "default",
+        "properties": {
+		  "properties": {
```
Comment (TianoMS) [inline tests/2018-03-01-preview/Microsoft.Storage.tests.json]: >		 [](start = 0, length = 2)

please use spaces instead of tabs.

**Comment by Tiano2017** [comment]
please fix a small issue on format. will merge PR once it's fixed.

**Comment by blueww** [comment]
@TianoMS  @Tiano2017 

I have resolved the comments from you, and updated the test per a new sample from feature PM.
Please help to review and merge.

File: vim.exe.stackdump [hunk 2]
```diff
@@ -0,0 +1,19 @@
+Stack trace:
```
Comment (TianoMS) [inline vim.exe.stackdump]: please remove this file..

File: tests/2018-03-01-preview/Microsoft.Storage.tests.json [hunk 3]
```diff
@@ -0,0 +1,186 @@
+{
+  "tests": [
```
Comment (TianoMS) [inline tests/2018-03-01-preview/Microsoft.Storage.tests.json]: >tests [](start = 3, length = 5)

some test cases are failing. please fix them.

Comment (blueww) [reply tests/2018-03-01-preview/Microsoft.Storage.tests.json]: @TianoMS 

I see all checks pas below. And I have run the test on my local machine and all pass.
Would you please show how to repro the test failure?
And what's the test failure message?

Thanks!

Comment (Tiano2017) [reply tests/2018-03-01-preview/Microsoft.Storage.tests.json]: my bad. the test cases are passing.

Comment (blueww) [reply tests/2018-03-01-preview/Microsoft.Storage.tests.json]: @TianoMS 
Could you give more details for the test failure? like how to repro, and the test failure message?
It seems I can't repro.

Comment (Tiano2017) [reply tests/2018-03-01-preview/Microsoft.Storage.tests.json]: There's no test failure.. I was wrong. Something went wrong when I merge with your branch. So you are all good.

**Comment by blueww** [comment]
@TianoMS 
Thanks for the review and merge! 👍

**Pull request merged**